### ISSUE TITLE
VideoCaptureStream

### ISSUE THREAD

**Issue by bertelschmitt** [chenxinfeng4/ffmpegcv#19]
Did a quick test

```
import ffmpegcv as fcv
cap = fcv.VideoCaptureStream("rtsp://admin:admin@123.456.7.89:554/stream1", cv2.CAP_FFMPEG)
```

gives:

 ```
 File "/usr/local/bin/.virtualenvs/catdetector/lib/python3.8/site-packages/ffmpegcv/__init__.py", line 398, in VideoCaptureStream
    return FFmpegReaderStream.VideoReader(
  File "/usr/local/bin/.virtualenvs/catdetector/lib/python3.8/site-packages/ffmpegcv/ffmpeg_reader_stream.py", line 20, in VideoReader
    assert pix_fmt in ["rgb24", "bgr24", "yuv420p", "nv12", "gray"]
AssertionError
```

Stream is  720, x 1280 color  h264.  Works with cv2
`cap = cv2.VideoCapture("rtsp://admin:admin@123.456.7.89:554/stream1")`

**Comment by chenxinfeng4** [comment]
It should be.
```
cap = fcv.VideoCaptureStream("rtsp://admin:admin@123.456.7.89:554/stream1")
```

```
# ffmpegcv
import ffmpegcv
cap = ffmpegcv.VideoCaptureStream(stream_url)
while True:
    ret, frame = cap.read()
    if not ret:
        break
    pass
```

**Comment by chenxinfeng4** [comment]
The Stream Reader is still experimental. Be very careful. I recommand you to use opencv.

**Comment by bertelschmitt** [comment]
Let's not give up so easily. I did a few more tests with ffmpegcv.VideoCaptureStream.

```
url = " rtsp://admin:admin@192.168.1.31:554/stream1”
cap =  fmpegcv.VideoCaptureStream(url)
```

That call hangs.  I traced it a bit further, and the problem appears to be in line 8 of stream_info.py:

`output = subprocess.check_output(cmd, shell=True)`

Execution of that line hangs forever. We probably should not rely on the call returning. I am no ffmpeg expert (the book is ordered and will arrive next week.) All I know is that shell=True should be avoided, but that’s probably beside the point here
Standing by for further tests.
